requirements.txt

```diff
@@ -1,5 +1,8 @@
 Gymnasium==0.26.3
 Pillow>=9.0.0
 selenium>=4.5.0
-pytest>=7.0.0
 numpy>=1.18.0
+
```

Comment:
This is the incorrect style for requirements.txt
We could switch over to pyproject.toml and remove the requirements.txt

Reply:
I removed requirements.txt since the dependencies are already in setup.py.